src/model.ts

```diff
@@ -1,3 +1,44 @@
-export interface User {
-  name: string;
+export interface ValidationResult {
+  key?: string;
+  type: string;
+  succeeded: boolean;
+  errorMessage: string;
```

Comment:
When we use the library in react-final-forms, we get something like:
`meta.error.errorMessage`.

Maybe we could rename to `message`.  What do you think?

Reply:
Good point, renamed